enter.pollinations.ai/test/mocks/snapshots/image-9dea1edc159ccdf3ab5e619a64a2d769.json

```diff
@@ -0,0 +1,40 @@
00\u0000\u0002#minf\u0000\u0000\u0000\u0014vmhd\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000$dinf\u0000\u0000\u0000\u001cdref\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\furl \u0000\u0000\u0000\u0001\u0000\u0000\u0001\u00e3stbl\u0000\u0000\u0000\u00bfstsd\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u00afavc1\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000@\u0000@\u0000H\u0000\u0000\u0000H\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0015Lavc62.11.100 libx264\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0018\u00ff\u00ff\u0000\u0000\u00005avcC\u0001d\u0000\n\u00ff\u00e1\u0000\u0018gd\u0000\n\u00ac\u00d9D&\u00c0D\u0000\u0000\u0003\u0000\u0004\u0000\u0000\u0003\u0000\u00c8<H\u0096X\u0001\u0000\u0006h\u00eb\u00e3\u00cb\"\u00c0\u00fd\u00f8\u00f8\u0000\u0000\u0000\u0000\u0010pasp\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0014btrt\u0000\u0000\u0000\u0000\u0000\u00005\u009b\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0018stts\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\r\u0000\u0000\u0002\u0000\u0000\u0000\u0000\u0014stss\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0001\u0000\u0000\u0000xctts\u0000\u0000\u0000\u0000\u0000\u0000\u0000\r\u0000\u0000\u0000\u0001\u0000\u0000\u0004\u0000\u0000\u0000\u0000\u0001\u0000\u0000\n\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0004\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0002\u0000\u0000\u0000\u0000\u0001\u0000\u0000\n\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0004\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0002\u0000\u0000\u0000\u0000\u0001\u0000\u0000\n\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0004\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0002\u0000\u0000\u0000\u0000\u001cstsc\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0001\u0000\u0000\u0000\r\u0000\u0000\u0000\u0001\u0000\u0000\u0000Hstsz\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\r\u0000\u0000\u0002\u00d6\u0000\u0000\u0000\u000e\u0000\u0000\u0000\f\u0000\u0000\u0000\f\u0000\u0000\u0000\f\u0000\u0000\u0000\u0014\u0000\u0000\u0000\u000e\u0000\u0000\u0000\f\u0000\u0000\u0000\f\u0000\u0000\u0000\u0014\u0000\u0000\u0000\u000e\u0000\u0000\u0000\f\u0000\u0000\u0000\f\u0000\u0000\u0000\u0014stco\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u00000\u0000\u0000\u0000audta\u0000\u0000\u0000Ymeta\u0000\u0000\u0000\u0000\u0000\u0000\u0000!hdlr\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000mdirappl\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000,ilst\u0000\u0000\u0000$\u00a9too\u0000\u0000\u0000\u001cdata\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000Lavf62.3.100",
```

Comment:
We should probably update the VCR to store binary streams in base64, i think storing it as a string could cause problems. Should be a quick addition as well.